origami.json

```diff
@@ -11,5 +11,8 @@
 		"optional": [
 			"matchmedia"
 		]
+	},
+	"ci": {
+		"circle": "https://circleci.com/api/v1/project/Financial-Times/o-viewport"
```

Comment:
The [Origami JSON spec](http://origami.ft.com/docs/syntax/origamijson/#format) doesn't mention a `circle` property. Maybe we should open a PR to update the spec page and allow it?